### DISCUSSION TITLE
connection between containers behind glutun

### DISCUSSION THREAD

**Discussion by steff2632** [qdm12/gluetun#673] [Q&A] [answered]
I have two containers behind gluetun and I can't seem to get them to communicate with each other. I can control them from another machine using 192.168.x.x:port addresses but communication between them using those addresses doesn't work. Any ideas?

**Comment by qdm12** [comment, marked as answer]
Use localhost or `127.0.0.1`. Technically they're the same host 😉

So e. g. `127.0.0.1:81234`

**Reply by jasongordon** [reply]
This should be in the wiki somewhere

**Reply by qdm12** [reply]
Added in https://github.com/qdm12/gluetun/wiki/Inter-containers-networking

**Comment by steff2632** [comment]
ahh, thanks, I had tried that but was using the "external" port rather than the "internal" one.

Curious, how would a container inside the vpn network talk to something in the 192.168.x.x range?

**Reply by qdm12** [reply]
Try setting `FIREWALL_OUTBOUND_SUBNETS=192.168.1.0/24` (From the top of my head check the wiki if that's not the one).
